sdk/storage/Azure.Storage.Files/src/Models/StorageDirectoryInfo.cs

```diff
@@ -38,4 +38,16 @@ internal StorageDirectoryInfo(RawStorageDirectoryInfo rawStorageDirectoryInfo)
             this.SmbProperties = new FileSmbProperties(rawStorageDirectoryInfo);
         }
     }
+
+    /// <summary>
+    /// FilesModelFactory provides utilities for mocking.
+    /// </summary>
+    public static partial class FilesModelFactory
+    {
+        /// <summary>
+        /// Creates a new StorageDirectoryInfo instance for mocking.
+        /// </summary>
+        public static StorageDirectoryInfo StorageDirectoryInfo(RawStorageDirectoryInfo rawStorageDirectoryInfo)
```

Comment:
`RawStorageDirectoryInfo` and all the other `Raw` classes need to be internal.

Reply:
The Raw* classes were public before this PR, and we open an issue to fix this outside of this PR?

Reply:
They were public because we weren't wrapping them.  It's really trivial to fix this now - you just set "x-ms-public" to false in the README everywhere you're renaming something to Raw.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/7390